packages/containers/src/index.js

```diff
@@ -35,6 +36,8 @@ import SubHeaderBar from './SubHeaderBar';
 import Typeahead from './Typeahead';
 import TabBar from './TabBar';
 
+const Layout = cmfConnect({})(PureLayout);
```

Comment:
Do you really need that ?
The containers.registerAllContainers register components and containers.
If it is not exposed by containers, it takes the component and cmfConnect if. Let not expose the pure component here.

Reply:
yes but its a breaking change. I add a comment on it